generic_tracers/generic_tracer.F90

```diff
@@ -141,7 +141,7 @@ module generic_tracer
   logical :: do_vertfill_post = .false.
   logical :: generic_tracer_register_called = .false.
   logical :: force_update_fluxes = .false.
-  character(len=10) :: as_param   = 'gfdl_cmip6'     ! Use default Wanninkhoff/OCMIP2 parameters for air-sea gas transfer
+  character(len=10) :: as_param   = 'W14'     ! Use Wanninkhoff 2014 parameters for air-sea gas transfer by default
```

Comment:
That's a good point I forgot, changing the default here will actually change the default for every generic tracer that uses the air-sea parameters. I think this is desirable, since the old default gfdl_cmip6 uses a mix of W92 and W14. Setting the namelist `as_param = 'gfdl_cmip6` would be needed to reproduce old answers.